### DISCUSSION TITLE
Example 2 of Invoke-Pester doesn't work

### DISCUSSION THREAD

**Discussion by DarkLite1** [PowerShellOrg/Plaster#400] [Documentation]
I found the `Plaster` module in the `vscode` module folder and wanted to check it out. After reading the examples and trying them out it appears that [example 2](https://github.com/PowerShell/Plaster/blob/master/docs/en-US/Invoke-Plaster.md) doesn't work:

`Invoke-Plaster -TemplatePath . -Destination ~\GitHub\NewModule -ModuleName Foo -Version 1.0.0`

> Invoke-Plaster : A parameter cannot be found that matches parameter name 'ModuleName'.

**Comment by alaurie** [comment]
I've put in a PR editing the documentation to explain that those parameters are dynamic and only valid if the imported plaster manifest contains them.

**Comment by fullenw1** [comment]
Replacing the error message with another like this one would be helpful too:

`The 'ModuleName' parameter cannot be used because it has not been defined in the parameter section of the plaster manifest`
